packages/site/cms-site/src/index.ts

```diff
@@ -8,6 +8,7 @@ export { InternalLinkBlock } from "./blocks/InternalLinkBlock";
 export { PixelImageBlock } from "./blocks/PixelImageBlock";
 export type { PropsWithData } from "./blocks/PropsWithData";
 export { SvgImageBlock } from "./blocks/SvgImageBlock";
+export { default as YouTubeVideoBlock } from "./blocks/YouTubeVideoBlock";
```

Comment:
Please don't use a default export, but a named export instead. You can use withPreview in combination with a named export, see SvgImageBlock for an example.